src/hooks/usePlaceOrder.ts

```diff
@@ -74,30 +74,42 @@ export const usePlaceOrder = (): Result => {
           exchangeApi.getCurrentBatchId(networkId),
         ])
 
-        if (sellTokenId === buyTokenId) {
-          toast.error('Cannot place order. Tokens are the same')
+        if (sellTokenId !== 0 || buyTokenId !== 0) {
+          log('sellTokenId, buyTokenId, batchId', sellTokenId, buyTokenId, batchId, sellToken.address, buyToken.address)
```

Comment:
Careful there with the merge. This is old code before the unregistered token fix!

Reply:
ah wtf? i thought sth weird happened, I'll check